deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,381 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "crdb.cockroachlabs.com/v1alpha1",
+          "kind": "CrdbCluster",
+          "metadata": {
+            "name": "crdb-tls-example"
+          },
+          "spec": {
+            "cockroachDBVersion": "v21.1.1",
+            "dataStore": {
+              "pvc": {
+                "spec": {
+                  "accessModes": [
+                    "ReadWriteOnce"
+                  ],
+                  "resources": {
+                    "requests": {
+                      "storage": "10Gi"
+                    }
+                  },
+                  "volumeMode": "Filesystem"
+                }
+              }
+            },
+            "nodes": 3,
+            "tlsEnabled": true
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: Database
+    certified: "true"
+    containerImage: registry.connect.redhat.com/cockroachdb/cockroachdb-operator:v1.7.8
+    createdAt: "2021-05-28T11:21:52Z"
+    description: CockroachDB Operator
+    operators.operatorframework.io/builder: operator-sdk-v1.1.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    repository: https://github.com/cockroachdb/cockroach-operator
+    support: Cockroach Labs
+  name: cockroach-operator.v1.7.8
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: CrdbCluster is the CRD for the cockroachDB clusters API
+      displayName: CockroachDB Operator
+      kind: CrdbCluster
+      name: crdbclusters.crdb.cockroachlabs.com
+      specDescriptors:
+      - description: (Required) Database disk storage configuration
+        displayName: Data Store
+        path: dataStore
+      - description: 'SupportsAutoResize marks that a PVC will resize without restarting
+          the entire cluster Default: false'
+        displayName: PVC Supports Auto Resizing
+        path: dataStore.supportsAutoResize
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      - description: (Required) Container image information
```

Comment:
Why is this described as required when it's optional?  It's only required if cockroachDBVersion isn't set